spdk.spec

```diff
@@ -141,27 +140,9 @@ make -C doc
 %make_install %{?_smp_mflags} prefix=%{_prefix} libdir=%{_libdir} datadir=%{_datadir}
 
 # Install tools
-mkdir -p %{install_datadir}
-find scripts -type f -regextype egrep -regex '.*(spdkcli|rpc).*[.]py' \
-	-exec cp --parents -t %{install_datadir} {} ";"
+mkdir -p %{install_datadir}/scripts
 
-# env is banned - replace '/usr/bin/env anything' with '/usr/bin/anything'
-find %{install_datadir}/scripts -type f -regextype egrep -regex '.*([.]py|[.]sh)' \
-	-exec sed -i -E '1s@#!/usr/bin/env (.*)@#!/usr/bin/\1@' {} +
-
-%if "%{use_python2}" == "1"
-find %{install_datadir}/scripts -type f -regextype egrep -regex '.*([.]py)' \
-	-exec sed -i -E '1s@#!/usr/bin/python3@#!/usr/bin/python2@' {} +
-%endif
```

Comment:
Why remove this?  If the RPM is being built to use python2 rather than python3, then this needs to be done.  If it's not, then this should be ignored.  IOW, the `%{use_python2}` conditional here should be what decides if this is done or not.

Reply:
we don't need the scripts, they aren't being used and were causing build errors

Reply:
It would probably be better to comment these lines out then, so that somebody in the future needing to resurrect any, knows that they need to be fixed also.

Reply:
done